apps/deploy-web/src/pages/api/config.ts

```diff
@@ -0,0 +1,18 @@
+import { LoggerService } from "@akashnetwork/logging";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { serverEnvConfig } from "@src/config/server-env.config";
+
+const logger = LoggerService.forContext("api/config");
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (!serverEnvConfig.UI_TESTS_TOKEN && serverEnvConfig.NEXT_PUBLIC_TURNSTILE_ENABLED) {
+    logger.warn(`UI_TESTS_TOKEN is not set, but turnstile protection is enabled.` + `It means UI tests will not be able to pass the turnstile protection.`);
```

Comment:
it probably doesn't matter much but this exists for tests debugging purposes only, however we'll have a lot of these logs in production where it's redundant.

Reply:
makes sense. I thought that we will apply the same pattern to both prod and beta. sooner or later we will want to run test on prod, at least on prod-sandbox